app/helpers/podcast_metrics_helper.rb

```diff
@@ -16,81 +12,49 @@ def sum_rollups(rollups)
     rollups.map { |r| r[:count] }.reduce(:+)
   end
 
-  def interval_options(date_start, date_end)
-    day_count = (date_start.to_date..date_end.to_date).to_a.length
-    if day_count > 60
-      [
-        ["Daily", "DAY"],
-        ["Weekly", "WEEK"],
-        ["Monthly", "MONTH"]
-      ]
-    elsif day_count < 30
-      [
-        ["Hourly", "HOUR"],
-        ["Daily", "DAY"],
-        ["Weekly", "WEEK"]
-      ]
-    else
-      [
-        ["Hourly", "HOUR"],
-        ["Daily", "DAY"],
-        ["Weekly", "WEEK"],
-        ["Monthly", "MONTH"]
-      ]
-    end
+  def interval_options
+    [
+      ["Hourly", "HOUR"],
+      ["Daily", "DAY"],
+      ["Weekly", "WEEK"],
+      ["Monthly", "MONTH"]
+    ]
   end
 
-  def rollups_date_range_options
+  def date_range_options
     [
-      ["Last 7 Days", 7.days.ago.utc.to_date],
-      ["Week to Date", Time.zone.now.utc.beginning_of_week.to_date],
-      ["Last 14 Days", 14.days.ago.utc.to_date],
-      ["Last 28 Days", 28.days.ago.utc.to_date],
-      ["Last Month", 1.month.ago.utc.to_date],
-      ["Month to Date", Time.zone.now.utc.beginning_of_month.to_date],
-      ["Last 3 Months", 3.months.ago.utc.to_date],
-      ["Last 6 Months", 6.months.ago.utc.to_date],
-      ["Last Year", 1.year.ago.utc.to_date],
-      ["Year to Date", Time.zone.now.utc.beginning_of_year.to_date]
+      ["Last 7 Days", [7.days.ago.utc.to_date, Time.zone.now.utc.to_date].to_json],
+      ["Week to Date", [Time.zone.now.utc.beginning_of_week.to_date, Time.zone.now.utc.to_date].to_json],
+      ["Last 14 Days", [14.days.ago.utc.to_date, Time.zone.now.utc.to_date].to_json],
+      ["Last 28 Days", [28.days.ago.utc.to_date, Time.zone.now.utc.to_date].to_json],
+      ["Last Month", [1.month.ago.utc.to_date, Time.zone.now.utc.to_date].to_json],
+      ["Month to Date", [Time.zone.now.utc.beginning_of_month.to_date, Time.zone.now.utc.to_date].to_json],
+      ["Last 3 Months", [3.months.ago.utc.to_date, Time.zone.now.utc.to_date].to_json],
+      ["Last 6 Months", [6.months.ago.utc.to_date, Time.zone.now.utc.to_date].to_json],
+      ["Last Year", [1.year.ago.utc.to_date, Time.zone.now.utc.to_date].to_json],
+      ["Year to Date", [Time.zone.now.utc.beginning_of_year.to_date, Time.zone.now.utc.to_date].to_json]
```

Comment:
So I think all these `.zone`s are unnecessary? I might suggest DRY-ing this up a bit by:

- Throwing the utc_today used all over the place as a var up top
- Or do something like I did [in spots](https://github.com/PRX/dovetail-spots/blob/main/config/initializers/extensions.rb#L27), and add a `Date.utc_today` method
  - Then you can change things like `Time.zone.now.utc.beginning_of_week.to_date` into `utc_today.beginning_of_year` or `Date.utc_today.beginning_of_year`
- Do a `.map` over this thing to do the `.to_json` calls from every line.  And maybe to add the 2nd param to them all, which seems to always be `utc_today`.

Would also be nice to translate these at some point.  But I'm okay if all the i18n stuff comes later.